src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPool.Windows.cs

```diff
@@ -13,6 +13,9 @@ public static partial class ThreadPool
         internal static bool UseWindowsThreadPool { get; } =
             AppContextConfigHelper.GetBooleanConfig("System.Threading.ThreadPool.UseWindowsThreadPool", "DOTNET_ThreadPool_UseWindowsThreadPool");
 
+        // Exposes whether or not the Windows thread pool is used for diagnostics purposes
+        private static readonly bool s_useWindowsThreadPoolForDebugger = UseWindowsThreadPool;
```

Comment:
```suggestion
        private static readonly bool s_useWindowsThreadPool = AppContextConfigHelper.GetBooleanConfig("System.Threading.ThreadPool.UseWindowsThreadPool", "DOTNET_ThreadPool_UseWindowsThreadPool");
        
        internal static bool UseWindowsThreadPool => s_useWindowsThreadPool;
```
?

Reply:
I think the field should reflect what the property returns because the property can be stubbed by trimming, such that sos reflects the actual state of which thread pool is being used and not just the config value.

Reply:
I don't know if coreclr can use trimming currently, and maybe it's not relevant currently, but I suppose it doesn't hurt to assume that either coreclr could use trimming in the future or an sos for NativeAOT could exist in the future and show similar info.